File: templates/csv.py
"""
TITLE.py

Script to ...

"""

# Load libraries
import __main__ as main
import sys
import pandas as pd
from dotenv import load_dotenv

# Load environment variables from .env file
load_dotenv(dotenv_path=".env")

# Define args interactively or accept commandArgs
if not hasattr(main, "__file__"):
    argv = ["code",
            "input",
            "output"
    ]
else:
    argv = sys.argv

# -- Process data here --


# Save csv file
res.to_csv(argv[-1])

File: templates/csv.R
# Script to ...

# Load libraries
suppressPackageStartupMessages({
  require(tidyverse)
})

# Load environment variables from .env file
dotenv::load_dot_env(file = ".env")

# Source modules
# source('file')

# Define args interactively or accept commandArgs
if(interactive()){
  .args <-  c('input',
              'output')
} else {
  .args <- commandArgs(trailingOnly = T)
}

# -- Process data here --


# Save csv result
write_csv(res, tail(.args, 1))
